sfcli/SmartFace.Cli/Commands/SubWatchlistMember/RegisterWatchlistMemberCmd.cs

```diff
@@ -30,15 +31,15 @@ public RegisterWatchlistMemberCmd(IWatchlistMemberRegistrationManager manager)
             Manager = manager;
         }
 
-        protected virtual int OnExecute(CommandLineApplication app, IConsole console)
+        protected virtual async Task<int> OnExecuteAsync(CommandLineApplication app, IConsole console)
         {
             var data = new RegisterWatchlistMemberExtended
             {
                 ExternalId = ExternalId,
                 PhotoFiles = Photos,
                 WatchlistExternalIds = WatchlistExternalIds
             };
-            Manager.RegisterWatchlistMember(data);
+            await Manager.RegisterWatchlistMemberAsync(data);
```

Comment:
Maybe rename _Manager_ to _RegistrationManager_ so it's clearer what is happening

Reply:
done